doc/multiple_cursors.txt

```diff
@@ -83,52 +101,16 @@ plugin taking over your favorite key bindings, you can turn off the default
 with >
 
     let g:multi_cursor_use_default_mapping=0
-<
-
-*g:multi_cursor_next_key* (Default: '<C-n>')
-*g:multi_cursor_prev_key* (Default: '<C-p>')
-*g:multi_cursor_skip_key* (Default: '<C-x>')
-*g:multi_cursor_quit_key* (Default: '<Esc>')
-You can map the 'next', 'previous', 'skip', and 'exit' keys like the
-following: >
 
     " Default mapping
+    let g:multi_cursor_start_word_key='<C-n>'
+    let g:multi_cursor_start_key='g<C-n>'
     let g:multi_cursor_next_key='<C-n>'
     let g:multi_cursor_prev_key='<C-p>'
     let g:multi_cursor_skip_key='<C-x>'
     let g:multi_cursor_quit_key='<Esc>'
 <
 
-*g:multi_cursor_start_key* (Default: 'g:multi_cursor_next_key')
-By default, the same key is used to enter multicursor mode as to select the
-next cursor location.  If you want to use a different key to start multicursor
-mode than for selecting the next location, do like the following: >
-
-    " Map start key separately from next key
-    let g:multi_cursor_start_key='<F6>'
-<
-
-*g:multi_cursor_start_word_key*
-When multicursor mode is started, it selects current word without
-boundaries, i.e. it behaves like `g*`. If you want to use word boundaries in
-Normal mode (as `*` does) but still have old behaviour up your sleeve, you can
-do the following: >
-
-    let g:multi_cursor_start_key='g<C-n>'
-    let g:multi_cursor_start_word_key='<C-n>'
-<
-
-In this configuration <C-n> will start multicursor mode using word boundaries
-(but only in Normal mode, as it does not make much sense to use it in Visual
-mode). Old behaviour without word boundaries is still available using
-g<C-n>.
-
-IMPORTANT: Please note that currently only single keystrokes and special
-keys can be mapped. This contraint is also the reason why multikey commands
-such as `ciw` do not work and cause unexpected behavior in Normal mode. This
```

Comment:
Sorry, I may have missed it, is this remark about `ciw` not true anymore?  Can I do `ciw` in Normal mode when multiple cursors are active?

Reply:
@balta2ar , yes, ```ciw``` works in normal mode.
Make sure to have the following in your **.vimrc**:
```vim
let g:multi_cursor_exit_from_insert_mode = 0
let g:multi_cursor_exit_from_visual_mode = 0
```
Then, ```<C-n>...<C-n><Esc>ciw``` will work fine :)